specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/preview/2020-09-01-preview/databoxedge.json

```diff
@@ -0,0 +1,8511 @@
+        },
+        "port": {
+          "format": "int32",
+          "description": "The port.",
+          "type": "integer",
+          "readOnly": true
+        }
+      }
+    },
+    "NetworkSettings": {
+      "description": "The network settings of a device.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ARMBaseModel"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/NetworkSettingsProperties",
+          "description": "The properties of network settings of a device.",
+          "readOnly": true,
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "NetworkSettingsProperties": {
+      "description": "The properties of network settings.",
+      "type": "object",
+      "properties": {
+        "networkAdapters": {
+          "description": "The network adapter list on the device.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkAdapter"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "Node": {
+      "description": "Represents a single node in a Data box Edge/Gateway device\r\nGateway devices, standalone Edge devices and a single node cluster Edge device will all have 1 node\r\nMulti-node Edge devices will have more than 1 nodes",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ARMBaseModel"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/NodeProperties",
+          "description": "The properties of the node",
+          "readOnly": true,
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "NodeInfo": {
+      "description": "Kubernetes node info",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Node name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Node type - Master/Worker",
+          "enum": [
+            "Invalid",
+            "Master",
+            "Worker"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "KubernetesNodeType",
+            "modelAsString": true
+          }
+        },
+        "ipConfiguration": {
+          "description": "IP Configuration of the Kubernetes node.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/KubernetesIPConfiguration"
+          }
+        }
+      }
+    },
+    "NodeList": {
+      "description": "Collection of Nodes.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of Nodes.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Node"
+          },
+          "readOnly": true
```

Comment:
Consider making this list pageable and adding a next link property

Reply:
Changed